x-pack/plugins/spaces/server/capabilities/capabilities_switcher.test.ts

```diff
@@ -317,4 +320,76 @@ describe('capabilitiesSwitcher', () => {
 
     expect(result).toEqual(expectedCapabilities);
   });
+
+  describe('when the space has a solution set', () => {
+    it('does toggles capabilities of the solutions different from the space one even when the space has no disabled features', async () => {
+      const space: Space = {
+        id: 'space',
+        name: '',
+        disabledFeatures: [],
+      };
+
+      const capabilities = buildCapabilities();
+
+      const { switcher } = setup(space);
+      const request = httpServerMock.createKibanaRequest();
+
+      {
+        space.solution = 'es';
+
+        // It should disable observability and securitySolution features
+        // which correspond to feature_2 and feature_3
+        const result = await switcher(request, capabilities, false);
+
+        const expectedCapabilities = buildCapabilities();
+
+        expectedCapabilities.navLinks.feature2 = false;
+        expectedCapabilities.catalogue.feature2Entry = false;
```

Comment:
I am not sure I fully understand why `feature2Entry` is set to `false` and not `feature3Entry`.